.github/workflows/ci.yml

```
@@ -247,16 +247,15 @@ jobs:
         run: yarn run build
 
       - name: pack
-        # Cannot use yarn lerna run pack
-        run: yarn lerna exec yarn pack
+        run: yarn workspaces foreach -A exec yarn pack
```

I think this needs topological ordering. Also consider using the long option names for readability (no need to abbreviate since nobody is typing it on the command line)
```suggestion
        run: yarn workspaces foreach --all --topological exec yarn pack
```